src/styles/Buttons.ts

```diff
@@ -12,27 +12,8 @@ const sharedDropDownBtnStyles = {
   marginTop: '0',
 };
 
-const sharedButtonStyles = {
-  padding: `${Spacing.xSmall}px ${Spacing.Medium}px !important`,
-  height: `${Spacing.xLarge}px`,
-  margin: '0 !important',
-};
-
 //BUTTONS LIGHT
 export const ButtonsLightRaw = {
-  Primary: {
```

Comment:
This wasn't used anywhere anymore. It's easier to use `color="primary"` which will use the palette Carla defined in `protocolTheme`. From my testing the colors were the same, even hover and active, including in dark mode.

I tried to do the same for Secondary as well, but there the colors were different especially for hover and active, so I didn't want to include the change in this PR.